iot-hub/Samples/service/PnpServiceSamples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -20,39 +20,39 @@ public class TemperatureControllerSample
 
         private readonly ServiceClient _serviceClient;
         private readonly RegistryManager _registryManager;
-        private readonly string _digitalTwinId;
```

Comment:
It looks like these samples use the IoT hub "native" APIs to demonstrate how to operate on a pnp compatible device (RegistryManager APIs). In that sense, it does make sense to me to refer to this as simply a "device", rather than a "digital twin".

It would be worth discussing how we'd like to present the same device when we use the pnp specific endpoints (and the pnp specific DigitalTwinClient) to interact with it: https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/service/DigitalTwinClientSamples/TemperatureController/TemperatureControllerSample.cs#L24